x-pack/plugins/uptime/public/components/overview/alerts/translations.ts

```diff
@@ -99,6 +132,92 @@ export const ENTER_NUMBER_OF_TIME_UNITS = i18n.translate(
   }
 );
 
+export const ENTER_NUMBER_OF_TIME_UNITS_DESCRIPTION = i18n.translate(
+  'xpack.uptime.alerts.monitorStatus.timerangeValueField.expression',
+  {
+    defaultMessage: 'within',
+  }
+);
+
+export const ENTER_NUMBER_OF_TIME_UNITS_VALUE = (value: number) =>
+  i18n.translate('xpack.uptime.alerts.monitorStatus.timerangeValueField.value', {
+    defaultMessage: 'last {value}',
+    values: { value },
+  });
+
+export const ENTER_AVAILABILITY_RANGE_ENABLED = i18n.translate(
+  'xpack.uptime.alerts.monitorStatus.availability.isEnabledCheckbox.label',
+  {
+    defaultMessage: 'Availability enabled',
```

Comment:
Can we rename this `Availability check` to be more consistent with `Status check` and drop `enabled` since the checkbox makes it obvious this is a boolean?

Reply:
Fixed in https://github.com/elastic/kibana/pull/70284/commits/42e8cd3d1e64a4bb018a815dcec9138780e9b81d.